tests/cli_test_case.py

```diff
@@ -294,3 +295,15 @@ def tree_sorted(obj):
                 return obj
 
         self.assertEqual(tree_sorted(a), tree_sorted(b))
+
+    def assert_test_path_orderly_equal(self, a, b):
```

Comment:
Let's define the contract of this method. What exactly does this method assert?

I ask because I was expecting `a` and `b` are symmetric, but L308-309 doesn't work that way. For exampe, if `a` is empty list, this assertion passes no matter what `b` is.

I'm almost tempted to just test `a['events']['testPath']==b['events']['testPath']`; Is that not what we want to do?

Reply:
> I'm almost tempted to just test a['events']['testPath']==b['events']['testPath']; Is that not what we want to do?

I initially planed to do, but it's not possible because the value associated with the 'events' key is in random order.